src/fr/composants/groupe-de-boutons-radio/base.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: 'layouts/base.njk'
+github: https://github.com/cds-snc/gcds-components/tree/main/packages/web/src/components/gcds-radio-group
+figma: https://www.figma.com/file/mh2maMG2NBtk41k1O1UGHV/Canadian-Digital-Service%E2%80%A8---GC-Design-System?node-id=818%3A3759&t=ciEmm7GYyGAY73zZ-0
+permalink: false
+tags: ['radiogroupFR', 'header']
+---
+
+# Groupe de Boutons Radio <br>`<gcds-radio-group>`
```

Comment:
no cap letters for 'boutons' and 'radio'. Should read: `Groupe de boutons radio`